introduction.md

```diff
@@ -2,3 +2,12 @@ Hi, my name is Katie.
 
 My favorite Code School Path so far has been HTML & CSS's Course on Sass and Bootstrap.
 
+*HTML
+*CSS
+*Sass
+*Bootstrap
+*Javascript
+*JQuery
+*Angular
+*Git
+*Ember
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?